src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/PrisonerSearchService.kt

```diff
@@ -94,6 +95,14 @@ class PrisonerSearchService(
     return reasons + hdcReasonIfPresent
   }
 
+  fun getIS91Status(prisonNumber: String): Boolean {
+    val prisoners = prisonerSearchApiClient.searchPrisonersByNomisIds(listOf(prisonNumber))
+    if (prisoners.size != 1) {
+      error("Found ${prisoners.size} prisoners for: $prisonNumber")
```

Comment:
Not too sure we should just error as this would crash the service - unless we have a better way to handle this? Can see we do the same in the getIneligibilityReasons endpoint so it's fine.